ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/MacOSX.yml

```diff
@@ -162,6 +180,14 @@
   when: not pdfwriter.stat.exists
   tags: jck_tools
 
+- name: Uninstall libpng
```

Comment:
I'm `-1` on this as I don't want end-users who use our playbooks to have things removed from their machine. Suggest we do this as a one off on all of our machines unless there is a specific reason for this to be in the playbooks. If it has to be, let's tag it with `adoptopenjdk`.

Also bear in mind that [we have a bug](https://github.com/adoptium/temurin-build/issues/3509) which is causing libpng to be picked up even when we tell it not to, which is not ideal :-)

Reply:
(Now tagged with adoptopenjdk so objection removed but we should plan to remove this in the future if we can)